frodo-runtime/src/main/java/com/fernandocejas/frodo/internal/DebugLog.java

```diff
@@ -18,6 +21,28 @@
    * @param message The message you would like logged.
    */
   public void log(String tag, String message) {
-    Log.d(tag, message);
+    if (!runningJUnit())
```

Comment:
It didn't work when I tried it. It was my first approach, actually. But you can try it and let me know, maybe I missed something I don't know ;)